### DISCUSSION TITLE
"Executor reports task instance finished (success) although the task says its queued. (Info: None) Was the task killed externally?".

### DISCUSSION THREAD

**Discussion by Amanwa19** [apache/airflow#36064] [Q&A]
Hi All ,  We are using apache airflow2.6.3, and it is installed on amazon EC2 instances. The Python version is 3.8.8 and we are using LocalExecutor . 

We are getting error message as "Executor reports task instance finished (success) although the task says its queued. (Info: None) Was the task killed externally?".  
Requesting expertise from the community again .

Thank you

**Comment by Pooort** [comment]
The same issue.
